### ISSUE TITLE
Things to possibly change

### ISSUE THREAD

**Issue by CurryK-del** [kruug/manjarno#1]
While I'm not one to say, `oh you shouldn't ever let your SSL certs expire` it's been 5 years since that's been a thing, so I don't think that's really a very valid issue to have on the page, but that's just my opinion.

the whole RCE thing, apt also had an RCE in it up until 2019 <https://nsfocusglobal.com/Linux-apt-RCE-Vulnerability-Threat-Alert>, which as you know is a base ubuntu package, so I'm wondering if maybe it would be a good idea to create an overview of each distro's negatives.

But that would be a lot to maintain and just wanted to share the info cause I thought it was interesting!

here's something that happened that is _really_ fishy though! I think this would be really important to include though
[Reddit: r/ManjaroLinux/ Change of Treasurer for Manjaro](https://www.reddit.com/r/ManjaroLinux/comments/hwo33h/change_of_treasurer_for_manjaro_community_funds/)
[Reddit: r/Linux Change of Treasurer for Manjaro](https://www.reddit.com/r/linux/comments/hwoev3/change_of_treasurer_for_manjaro_community_funds/)
[Linux Reviews: Manjaro Slush Fund](https://linuxreviews.org/Manjaro_Linux_Lead_Developer_In_Hot_Waters_Over_Donation_Slush_Fund_For_Laptop_And_Personal_Items)

TL;DR: Manjaro removed the treasurer after he questioned about expenses. 
also, the lead dev also had the funds deposited directly into their personal account so that's... something lol

**Comment by Scattered-Faith** [comment]
I think this stuff would be good to add, @kruug, questionable money practices are always bad

**Comment by kruug** [comment]
Added in commit e1bbcbb.
